src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -734,7 +734,10 @@ def canDatasetGoToDisk(self, dataItem, isRelVal=False):
                 self.alertManagerAPI.sendAlert(alertName, alertSeverity, alertSummary, alertDescription, self.alertServiceName)
             raise
 
-        if dataTier in self.uConfig['tiers_to_DDM']['value']:
+        # Bypass unified configuration for RelVals
```

Comment:
@haozturk Hasan, do you want to bypass only the Unified configuration? Or we better bypass whatever configurations we have (microservice + unified + campaign) and make sure that RelVal output is always placed on Disk?

If the latter, I'd suggest to start this method (L710) with something like:
```
        if isRelVal:
            self.logger.info("RelVal output always go to Disk. Container name: %s", dataItem['Dataset'])
            return True

```

Reply:
It would be good to bypass everything @amaltaro  Let me update it. I did not consider microservice configuration since it's empty. But, it might be filled later, so, let's be cautious. Thanks for the comment.

Reply:
it's done @amaltaro  can you please review it again?